src/ast.rs

```diff
@@ -339,6 +339,7 @@ pub struct FunctionDecl {
     /// the boolean indicates whether to bind by-reference (true)
     pub usev: Vec<(bool, RcStr)>,
     pub ret_ref: bool,
+    pub ret_type: Option<RcStr>
```

Comment:
what should "Option<Ty>" look if the type could be nullable? (?\Foo\Bar).

should i build something like:
Option<Nullable<Ty>>